apps/frontend/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import vector from '../../assets/icons/vector.svg';
+import clsx from 'clsx';
+
+interface TagProps {
+  label: string;
+  isSelected?: boolean;
+  onClick: () => void;
+}
+
+export const Tag = ({ label, isSelected, onClick }: TagProps) => {
+    const [selected, setSelected] = React.useState<boolean>(isSelected || false);
+
+  const commonClasses = 'rounded-full font-gellix font-semibold text-xl leading-5 px-[1.875rem] py-[0.938rem] border ';
+
+  const handleClick = ()=> {
+    setSelected(selected => !selected);
+    onClick();
+  }
+
+  return (
+    <button className={clsx(commonClasses,{
+        'flex bg-midnight-blue text-white': selected === true,
+        'text-midnight-blue border-midnight-blue hover:bg-midnight-blue hover:text-white': selected === false
+    })} onClick={handleClick}>
+      {label}
+      {selected && <img className='pl-[1.125rem] pt-[0.313rem]' src={vector} alt="cancel" />}
```

Comment:
`pl-0.5` it is the same as `pl-[1.125rem]`
`pt-1.5` it is the same as `pt-[0.313rem]`
Same for the above `px-[1.875rem] py-[0.938rem]` try to find the closest match:
https://tailwindcss.com/docs/padding

Reply:
No `pl-0.5` is equal to `pl-[0.125rem]`  not `1.125rem` and pt-1.5 is equal to `pt-[0.375rem]` not `pt-[0.313rem]`.

Reply:
pl-5 then, the closest